.storybook/pages/Cad/Cad.tsx

```diff
@@ -16,208 +16,529 @@ import {
+  const [containers, setContainers] = useState({
+    'container-1': {
+      itemIds: ['item-1', 'item-2', 'item-3', 'item-4', 'item-5'],
+      emptyContent: container1EmptyContent(),
+      header: (
+        <DragDropContainerHeader>
+          <Toolbar className="u-margin-bottom-md" variant="bare">
+            <ToolbarItem>
+              <Heading as="h2" size="title-sm" variant="base">
+                Available projects
+              </Heading>
+            </ToolbarItem>
+            <ToolbarItem align="right">
+              <Button variant="icon">
+                <Icon name="add" purpose="decorative" />
+                Add project
               </Button>
-              <Button variant="primary">
-                Next <Icon name="arrow-narrow-right" purpose="decorative" />
+            </ToolbarItem>
+          </Toolbar>
+        </DragDropContainerHeader>
+      ),
+    },
+    'container-2': {
+      itemIds: [],
+      emptyContent: container2EmptyContent(),
+      header: (
+        <DragDropContainerHeader>
+          <Toolbar className="u-margin-bottom-md" variant="bare">
+            <ToolbarItem>
+              <Heading as="h2" size="title-sm" variant="base">
+                Planned projects
+              </Heading>
+            </ToolbarItem>
+            <ToolbarItem align="right">
+              <Button variant="icon">
+                <Icon name="add" purpose="decorative" />
+                Add project
               </Button>
-            </ButtonGroup>
-          }
-          title="Course Planner: History 6"
-        />
-      </LayoutContainer>
-    </Panel>
-    <LayoutContainer>
-      <Layout sidebar="wide" variant="right-sidebar">
-        <LayoutSection region="main">
-          <Panel>
-            <Heading as="h2" className="u-margin-bottom-lg" size="headline-sm">
-              Select projects for your History 6 plan
-            </Heading>
-            <Text as="div" className="u-margin-bottom-lg">
-              <p>
-                Make a plan so that you can stay connected to learning
-                objectives even as changes occur throughout the year.
-              </p>
-              <p>
-                Room for more instructional copy per Content Strategy lorem
-                ipsum dolor sit amet, consectetur adipiscing elit. Phasellus est
-                quam, consequat iaculis pretium accumsan, fringilla id ligula.
-              </p>
-            </Text>
+            </ToolbarItem>
+          </Toolbar>
+        </DragDropContainerHeader>
+      ),
+    },
+  });
+  const [items, setItems] = useState({
+    'item-1': {
+      title: 'Project #1',
+      children: (
+        <ProjectCard
+          behavior="draggable"
+          buttonDropdownItems={projectCardMenuItemsWithDelete()}
+          meta="12 days"
+          number={1}
+          numberAriaLabel="Project 1"
+          title="Longer project card title that wraps"
+        ></ProjectCard>
```

Comment:
Nit: can this be self-closing? Same for the other `ProjectCard`s of course.
```suggestion
        />
```